addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml

```diff
@@ -19,6 +19,9 @@
                      invisible="not warning_message">
                     <field name="warning_message"/>
                 </div>
+                <div class="alert alert-warning text-center" role="alert" invisible="not l10n_in_display_null_pan_tds_warning">
+                    Since the partner doesn’t have a PAN number, please apply TDS at the higher rate.
```

Comment:
```suggestion
                    Since the partner doesn’t have a PAN, please apply TDS at the higher rate.
```
PAN (Permanent Account **Number**)